stubs/fpdf2/fpdf/drawing.pyi

```diff
@@ -1,18 +1,20 @@
 import decimal
 from _typeshed import Incomplete, Self
 from collections import OrderedDict
-from collections.abc import Generator
-from contextlib import _GeneratorContextManager
+from collections.abc import Generator, Iterator
+from contextlib import contextmanager
 from re import Pattern
-from typing import ClassVar, NamedTuple
+from typing import ClassVar, NamedTuple, TypeVar
 from typing_extensions import TypeAlias
 
 from .syntax import Name, Raw
 
 __pdoc__: dict[str, bool]
 
-def force_nodocument(item): ...
-def force_document(item): ...
+_T = TypeVar("_T")
```

Comment:
The docstring says "class or method. We shouldn't just allow everything: if you somehow end up doing `force_nodocument("blah blah")`, that will fail. Because classes and methods are both callable, we can restrict `_T` like this:

```suggestion
_T = TypeVar("_T", bound=Callable[Any])
```

You will need to import `Callable` and `Any` from `typing`.